resources/validations/src/ssp.sch

```diff
@@ -536,5 +536,30 @@
             <sch:value-of select="if (count($extraneous-parties)=1) then ' party' else ' parties'" />is not a defined party: 
             <sch:value-of select="$extraneous-parties/o:party-uuid" /></sch:assert>
         </sch:rule>
+        <sch:rule context="/o:system-security-plan/o:back-matter/o:resource">
+            <sch:assert id="resource-uuid-required"
+                        organizational-id="section-b.?????"
+                        test="./@uuid">[Section B Check ????] This SSP includes back-matter resource missing a UUID</sch:assert>
+        </sch:rule>
+        <sch:rule context="/o:system-security-plan/o:back-matter/o:resource/o:rlink">
+            <sch:assert id="resource-rlink-required"
+                        organizational-id="section-b.?????"
+                        test="doc-available(./@href)">[Section B Check ????] This SSP references back-matter resource: 
```

Comment:
I guess we need to have a talk about the best description for `[Section B Check ????]`, correct?